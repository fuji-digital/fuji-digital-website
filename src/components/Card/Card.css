.card {
  background-color: #291458;
  border-radius: 1.5rem;
  padding: 1.5rem 2rem;
  @media screen and (min-width: 768px) {
    padding: 3rem;
  }
}
