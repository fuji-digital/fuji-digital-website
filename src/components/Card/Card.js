import { StyledCard } from "./Card.styles";

export const Card = ({ children }) => {
  return <StyledCard>{children}</StyledCard>;
};
