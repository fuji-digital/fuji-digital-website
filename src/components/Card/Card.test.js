import renderer from "react-test-renderer";
import { Card } from "./Card";
import { render, screen } from "@testing-library/react";

describe("Card", () => {
  it("matches snapshot", () => {
    const child = <div>Hello</div>;
    const component = renderer.create(<Card children={child} />).toJSON();
    expect(component).toMatchSnapshot();
  });

  it("renders children", () => {
    render(
      <Card>
        <p>Child to find</p>
      </Card>
    );
    const child = screen.getByText("Child to find");
    expect(child).toBeInTheDocument();
  });
});
