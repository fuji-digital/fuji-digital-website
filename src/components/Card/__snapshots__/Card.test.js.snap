// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card matches snapshot 1`] = `
<div
  className="card"
>
  <div>
    Hello
  </div>
</div>
`;
